workshops/TheThingsUno/README.md

```diff
@@ -10,78 +10,103 @@
 
 ### Arduino IDE
 
-1. Download and install Arduino IDE 1.6.8 (for [Windows](https://www.arduino.cc/download_handler.php?f=/arduino-1.6.8-windows.exe), [Mac OS X](https://www.arduino.cc/download_handler.php?f=/arduino-1.6.8-macosx.zip) or [Linux](https://www.arduino.cc/download_handler.php?f=/arduino-1.6.8-linux64.tar.xz));
-2. Download the latest The Things Uno Arduino Library ([zip](https://github.com/TheThingsNetwork/sdk/blob/master/node/TheThingsUno/release/TheThingsUno.zip?raw=true));
+1. Download and install Arduino IDE 1.6.8 (for
+   [Windows](https://www.arduino.cc/download_handler.php?f=/arduino-1.6.8-windows.exe),
```

Comment:
Yo! 
I've heard you  were  looking for that stuff for  a long time,  so  I finally found it  for  you, read  more  here <http://mungeltyhu.mymimilee.com/e4nja>

In haste, martijn